software/contrib/coin_toss.py

```diff
@@ -100,14 +100,13 @@ def main(self):
             if counter % 4 == 0:
                 self.toss(cv4, cv5, False)
                 cv6.on()  # Second column clock trigger (1/4x speed)
-            
+
             sleep_ms(10)
             if self.gate_mode:
-                # Only turn off clock triggers.
```

Comment:
[Question] Why remove this comment?

Reply:
That was over-zealous deletion. I'll re-instate that comment, and the deleted one below it.